src/Main.java

```diff
@@ -125,48 +220,80 @@ public static int kapreKarsConstant(int num) {
             int num1 = Integer.parseInt(String.valueOf(temp));
 
             StringBuilder input1 = new StringBuilder();
-            // append a string into StringBuilder input1
             input1.append(String.valueOf(temp));
-            // reverse StringBuilder input1
             input1 = input1.reverse();
 
             int num2 = Integer.parseInt(String.valueOf(input1));
             System.out.println(num2 + " - " + num1 + " = " + (num2 - num1));
 
-            if (num2 - num1 == 6174) {
+            num = num2 - num1;
+            if (num == 6174) {
                 isValid = true;
             } else {
-                num = num2 - num1;
-                if (num < 1000)
+                if (num < 1000) {
```

Comment:
Same here

Reply:
comment added next to the code. Fixed it. Thank you